apps/condo/domains/common/utils/serverSchema/AnalyticsQueryBuilder.js

```diff
@@ -0,0 +1,57 @@
+const has = require('lodash/has')
+
+class AnalyticsQueryBuilder {
```

Comment:
Please, write comments about what this class does in common and provide usage examples.

Reply:
@AntonAL added some docstrings. Could you check this?